src/driver/test_point_aggregator.py

```diff
@@ -124,7 +124,21 @@ def aggregate(
     Returns:
       A single row DataFrame representing the values of the statistics listed in keys.
     """
-    stats = {}
+    stats = {"model_succeeded": [1], "model_exception": [""]}
     for key in AGGREGATORS:
         stats[key] = [AGGREGATORS[key](true_reach, simulated_reach)]
     return pd.DataFrame(data=stats)
+
+
+def aggregate_on_failure(inst: Exception) -> pd.DataFrame:
```

Comment:
How about calling it `aggregate_on_exception`?  

The essential question: is there any failure (model_succeeded = 0) but non-exception cases? I believe you've thought of them.

Two possible cases off the top of my head: (1) all training points have too small or even negative reach so that the fitted model is not meaningful (2) The optimizer raises a serious warning so that we know the optimization doesn't converge well.

Reply:
Yes, that's a better name.